kin-base/src/test/java/kin/base/ServerTest.java

```diff
@@ -109,6 +106,22 @@ public void testSubmitTransactionSuccess() throws IOException {
     assertNull(response.getExtras());
   }
 
+  @Test
+
+  public void test_ResponseCodeHttp307_SubmitTransactionSuccess() throws IOException {
+    mockWebServer.enqueue(
```

Comment:
something's not click for me here, I would expect 2 consecutive response for 2 expected requests, the first will be 307 **without** body, and then another one with 200 status and body.
that because the transaction shouldn't get the response from the first 307 response, it should avoid it and send a new request again.

Reply:
fixed